spec/lib/nationbuilder/client_spec.rb

```diff
@@ -50,6 +50,7 @@
     it 'should handle a parametered GET' do
       VCR.use_cassette('parametered_get') do
         response = client.call(:basic_pages, :index, site_slug: 'organizeralexandreschmitt', limit: 11)
+        expect(response['status_code']).to eq(200)
```

Comment:
Got it, so `delete` would continue to return `true`/`false` on any request that doesn't return `json` and the `status_code` would be returned in the parsed response otherwise. That works - though I feel this should be a minor bump to the version, not a patch (the next up would be `1.4.1` as `1.4.0` [was yanked](https://rubygems.org/gems/nationbuilder-rb/versions)).